spec/controllers/facility_reservations_controller_spec.rb

```diff
@@ -188,7 +188,7 @@
 
     context "redirect on no edit" do
       before :each do
-        @reservation.update_attribute(:canceled_at, Time.zone.now)
+        @reservation.order_detail.update_attributes(canceled_at: Time.zone.now)
```

Comment:
nbd, but I suspect we used `update_attribute` to be marginally quicker in the specs because we didn't care about validations. Does rubocop care about this? If so, does `update_column` also trigger the warning? I don't feel strongly about this.

Also, not caring too strongly, I believe we're leaning slightly in favour (all amatter of taste :)) of `Time.current` instead of `Time.zone.now`